Leave/leave_db.py

```diff
@@ -32,8 +32,35 @@ def GetLeaveStatus(request_id):
     leave = [dict(zip([column[0] for column in db.GetDBCursor().description], row)) for row in db.GetDBCursor().fetchall()][0]
     return leave["leave_status"]
 
-def GetEmergencyLeavesForYear(member_id, year):
-    db.GetDBCursor().execute(f"SELECT * FROM [leaves] WHERE member_id = {member_id} AND is_emergency = 'True' AND YEAR(date) = {year}")
+def GetPaidLeavesForYear(member_id, year, month = None):
+    query = f"SELECT * FROM [leaves] WHERE member_id = {member_id} AND leave_status = 'Approved' AND is_unpaid = 'False' AND YEAR(date) = {year}"
+    if month != None:
+        query += f' AND MONTH(date) = {month}'
```

Comment:
Why have this when the function is called `ForYear`

Reply:
i called it `ForYear` because this is the main param, but the month is optional which means you can input year and not month

Reply:
although i think that if we renamed the function to be called `GetX(member_id, year = None, month = None)` is descriptive enough and more generic what do you think ?

Reply:
Done